boards/xtensa/esp32/esp32_defconfig

```diff
@@ -32,3 +32,5 @@ CONFIG_I2C_1=y
 CONFIG_CLOCK_CONTROL=y
 
 CONFIG_BOOTLOADER_ESP_IDF=y
+CONFIG_LOG=y
+CONFIG_LOG_DETECT_MISSED_STRDUP=n
```

Comment:
I don't think these should be turned on by default.

Reply:
Removed as per suggestion